### DISCUSSION TITLE
[Help] Student not able to use SEB, log included

### DISCUSSION THREAD

**Discussion by jeroenhens** [SafeExamBrowser/seb-win-refactoring#246] [Q&A]
[2021-11-12_10h08m25s_Runtime.log](https://github.com/SafeExamBrowser/seb-win-refactoring/files/7538581/2021-11-12_10h08m25s_Runtime.log)

I have a student who can't use SEB. SEB is properly installed, but it keeps giving errors. I have the log included, she is using a laptop with no external screen, but it seems like SEB doesn't like the monitor configuration?

Anybody with a solution? Thx in advance!

**Comment by dbuechel** [comment]
There seems to be an issue on their system which, in case that they are not attempting to cheat, might be resolved by repairing or rebuilding the WMI repository: https://www.thewindowsclub.com/how-to-repair-or-rebuild-the-wmi-repository-on-windows-10

To workaround this issue, you could enable the setting "Ignore errors when validating display configuration" in your configuration: https://safeexambrowser.org/windows/win_usermanual_en.html#SecurityPane. This however will have the side effect that the display check doesn't help anymore with suppressing the usage of VMs.

**Reply by jeroenhens** [reply]
Thx, i'll give that a go!

**Reply by Noname1122Q** [reply]
Hello, dbuechel,
I am experiencing the same issue. I have repaired the WMI repository and it does come out as consistent as well, but the error is not resolved. Wondering if you could help.

**Reply by Noname1122Q** [reply]
Also here is some more additional info:

SEB worked before on the same PC + monitor setup, under a previous Windows installation.
SEB + Get-WmiObject -Class WmiMonitorID both fail with "Not supported".
The WMI class exists but querying it fails.
WMI repository reports "consistent".

**Reply by dbuechel** [reply]
> The WMI class exists but querying it fails.

That is the root issue then and needs to be resolved. You may also want to ensure that you've all the latest (including optional) Windows Updates as well as the correct and latest graphics card / display driver installed.

**Reply by Noname1122Q** [reply]
After more digging in this matter, I have some more information:

Have latest windows updates.
Have correct graphic drivers as well.
Also if I go to SEB Configuration Tool and check the option to Ignore display errors, I still get the same error.

i think i(in my case ) the issue has come because I initially installed a "superlite" version of windows which I updated later and it most likely did not have all necessary components for SEB to run.
I am installing the previous Windows OS that I used to run before and will update if the issue is resolved.

**Reply by Noname1122Q** [reply]
FIXED: SEB not launching / WMI ‘Not Supported’ error

After days of frustration, countless troubleshooting steps (including reinstalling Windows), the root cause turned out to be the most unexpected thing:
my “Premium” VGA cable.

This “better” cable was apparently blocking the monitor’s EDID data, which broke WMI monitor detection. Swapped it back to my old VGA cable — instantly fixed. SEB now launches perfectly

**Reply by dbuechel** [reply]
That is indeed also news to me, and unexpected indeed! Thanks a lot for sharing the solution, it may help other users with the same issue.
